### PR TITLE
Fix other links in main page

### PR DESCRIPTION
**Pull request by apraga**
JSServe -> Bonito
Tyler.jl updated

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -104,7 +104,7 @@ <h2 class="text-lg font-bold text-left" data-jscall-id="58">Surgical updates &#3
                 </div>
               </div>
               <div class="w-full md:basis-2/5" data-jscall-id="64">
-                <a class="visited:text-purple-600 text-blue-600" href="https://makieorg.github.io/Tyler.jl/dev/examples/generated/UserGuide/iceloss_ex" style="" target="_blank" data-jscall-id="65">
+                <a class="visited:text-purple-600 text-blue-600" href="https://makieorg.github.io/Tyler.jl/v0.1.4/iceloss_ex" style="" target="_blank" data-jscall-id="65">
```
Comment (asinghvi17) [inline docs/index.html:107]:
```suggestion
                <a class="visited:text-purple-600 text-blue-600" href="https://makieorg.github.io/Tyler.jl/stable/iceloss_ex" style="" target="_blank" data-jscall-id="65">

File: src/index.jl [hunk 2]
```diff
@@ -85,7 +85,7 @@ function index()
                 class="flex flex-col gap-1",
             );
             image="use_cases/glacier.mp4",
-            link="https://makieorg.github.io/Tyler.jl/dev/examples/generated/UserGuide/iceloss_ex",
+            link="https://makieorg.github.io/Tyler.jl/v0.1.4/iceloss_ex",
```
Comment (asinghvi17) [inline src/index.jl:88]:
```suggestion
            link="https://makieorg.github.io/Tyler.jl/stable/iceloss_ex",
```

**Review by asinghvi17 — CHANGES_REQUESTED** [review]

**Comment by apraga** [comment]
I've rebased onto the main branch to incorporate Simon's change.

**Comment by SimonDanisch** [comment]
Thank you!

**Pull request merged**